ImperatorToCK3/Source/CommonUtilities/History.h

```diff
@@ -0,0 +1,53 @@
+#ifndef HISTORY_H
+#define HISTORY_H
+
+
+#include "Parser.h"
+#include "Date.h"
+#include <map>
+
+
+
+struct SimpleFieldStruct {
```

Comment:
I tend to like to put every class and struct in its own file (so each file is very simple). You could put everything in a CommonUtilities/History is you choose to do this.